catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/SubmitStandingQueryRequestImplTest.java

```diff
@@ -378,7 +378,7 @@ private void setupStandingQueryRequest()
         QueryLifeSpan lifespan = getEmptyLifespan();
         NameValue[] properties = new NameValue[0];
         //Set artificially low for for test cases.
-        long defaultUpdateFrequencyMsec = 100;
+        long defaultUpdateFrequencyMsec = 2000;
```

Comment:
This is an increase in order to allow slower computers to update appropriately. A value of `100` was causing intermittent failures.